Logger.go

```diff
@@ -235,3 +251,17 @@ func actualValue(value interface{}) interface{} {
 
 	return value
 }
+
+const requestId = "x-request-id"
```

Comment:
de naam van een constante moet aangeven wat het is
```suggestion
const requestIdHeader = "x-request-id"
```